tests/ui_json_utils_test.py

```diff
@@ -298,6 +298,23 @@ def test_flatten_group_value():
     assert flat["test"] is None
 
 
+def test_range_label():
+    ui_json = deepcopy(default_ui_json)
```

Comment:
I was trying to get familiar with the new template and decided to create a points object with data and write out an input file with the new type in there.  Seems to be something wrong with the format:
![image](https://github.com/MiraGeoscience/geoh5py/assets/81254271/ada98ffc-3494-43dc-9209-bc1a96982997)
Here's the code that reproduces:
![image](https://github.com/MiraGeoscience/geoh5py/assets/81254271/09db0f41-ade7-4cfc-86d7-690a06c8c837)
Strangely, Analyst seems to handle it ok, we just can't turn around and read back in.  Am I missing something?  Is this expected?

Reply:
The problem doen't seems to come from the uijson but from "update_ui_values" in input_file (l.245)

It change the uijson.
Expected:
````
  "target": {
    "main": true,
    "label": "Threshold",
    "allowComplement": false,
    "isComplement": false,
    "parent": "grid_object",
    "property": "{e9567358-a3f9-4b46-8b5d-ce18ab4740b8}",
    "association": [
      "Cell",
      "Vertex"
    ],
    "dataType": [
      "Referenced",
      "Integer",
      "Float"
    ],
    "value": [
      0.49085091490850913,
      1.0
    ],
    "rangeLabel": "Values",
    "enabled": true
  },
````

obtained in the test:

````
    "test": {
        "main": true,
        "label": "Range",
        "allowComplement": false,
        "isComplement": false,
        "parent": "object",
        "property": "{bd70a785-7f51-48bc-9875-47afbcd879d6}",
        "association": "Vertex",
        "dataType": "Float",
        "value": {
            "property": "{bd70a785-7f51-48bc-9875-47afbcd879d6}",
            "value": [
                0.2,
                0.8
            ],
            "isComplement": false
        },
        "rangeLabel": "Values",
        "enabled": true
    }
````

Reply:
it warps up the value in a sub-dictionary. 
But I don't know why...

Reply:
I corrected it handling the exception for this widget.